src/marten/template/filter/linebreaks.cr

```diff
@@ -0,0 +1,14 @@
+module Marten
+  module Template
+    module Filter
+      # The `linebreaks` filter.
+      #
+      # The `linebreaks` filter allows to convert a string replacing all newlines with HTML line breaks (<br />).
+      class LineBreaks < Base
+        def apply(value : Value, arg : Value? = nil) : Value
+          Value.from(value.to_s.gsub("\n", "<br />"))
```

Comment:
In Marten templates, `String` objects are automatically [HTML-escaped](https://martenframework.com/docs/templates/introduction#auto-escaping). This means that with the present implementation, the `<br />` tags injected by this filter would be escaped by the Marten template engine automatically when rendering templates (eg. to `&lt;br /&gt;`).

To avoid this, you will need to convert the string to a "safe string" object in order to tell Marten to not automatically escape the string returned by the `linebreaks` filter. But by doing this we would also make the use of `linebreaks` vulnerable to Cross-Site Scripting (XSS) attacks because the content of `value` would be marked as "safe" and rendered as HTML as well. So we need to do two things:

1. HTML-escape the incoming `value` and replace the `\n` characters
2. Mark the resulting string as "safe"

This could be achieved with something like that:

```suggestion
          Value.from(SafeString.new(HTML.escape(value.to_s).gsub("\n", "<br />")))
```

Obviously, this will also involve some changes at the specs level! 🙂